docs/execute-methods.md

```diff
@@ -820,6 +820,14 @@ The response looks like `{"value":{"left":0,"top":96,"width":828,"height":1696}}
 
 `width` and `height` are the screen's width and height.
 
+### mobile: viewportScreenshot
```

Comment:
I have deliberately not documented this extension as it is unstable and does not guarantee proper viewport screenshot to be returned (I would love it to be either removed completely or fixed properly)

Reply:
It was already present in the autogenerated docs but with no real description. I can just add a warning that it is unstable?

Reply:
yes, that makes sense